entities/jackson/src/main/java/org/ccci/gto/globalreg/entity/jackson/Authentication.java

```diff
@@ -0,0 +1,233 @@
+
+package org.ccci.gto.globalreg.entity.jackson;
+
+import javax.annotation.Generated;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import org.apache.commons.lang.builder.EqualsBuilder;
+import org.apache.commons.lang.builder.HashCodeBuilder;
+import org.apache.commons.lang.builder.ToStringBuilder;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@Generated("org.jsonschema2pojo")
+@JsonPropertyOrder({
+    "id",
+    "key_guid",
+    "relay_guid",
+    "parent_id",
+    "created_by",
+    "updated_at",
+    "user_entered"
+})
+public class Authentication {
+
+    @JsonProperty("id")
+    private String id;
+    @JsonProperty("key_guid")
+    private KeyGuid keyGuid;
```

Comment:
Good observation.

The model was produced from `filters[owned_by]=all` query which apparently differs from SBR.

I had realized that I might expand the model to include more data.  I also considered that the GR might modify its model over time. However, I was unaware (until now) that the model could change depending on the query in question.

So, there are indeed more factors than I was aware that might result in clients models varying.

I can still see two of my client applications using the same SBR based model, if that makes it all worth continuing an attempt to centralize them.

Again, I'm open to your suggestions.